native/acter/api.rsh

```diff
@@ -1022,8 +1022,80 @@ object RoomEventItem {
     /// the type of massage, like text, image, audio, video, file etc
     fn msg_type() -> Option<string>;
 
-    /// covers text/image/audio/video/file/location/emote/sticker
-    fn msg_content() -> Option<MsgContent>;
+    /// covers text/image/audio/video/file/location/emote
+    fn message() -> Option<MsgContent>;
+
+    /// covers m.sticker
+    fn sticker() -> Option<Sticker>;
+
+    /// covers some of m.room.member
+    fn membership_change() -> Option<MembershipChange>;
+
+    /// covers some of m.room.member
+    fn profile_change() -> Option<ProfileChange>;
+
+    /// covers room state change
+    fn policy_rule_room() -> Option<PolicyRuleRoomContent>;
+
+    /// covers room state change
+    fn policy_rule_server() -> Option<PolicyRuleServerContent>;
+
+    /// covers room state change
+    fn policy_rule_user() -> Option<PolicyRuleUserContent>;
+
+    /// covers room state change
+    fn room_aliases() -> Option<RoomAliasesContent>;
+
+    /// covers room state change
+    fn room_avatar() -> Option<RoomAvatarContent>;
+
+    /// covers room state change
+    fn room_canonical_alias() -> Option<RoomCanonicalAliasContent>;
+
+    /// covers room state change
+    fn room_create() -> Option<RoomCreateContent>;
+
+    /// covers room state change
+    fn room_encryption() -> Option<RoomEncryptionContent>;
+
+    /// covers room state change
+    fn room_guest_access() -> Option<RoomGuestAccessContent>;
+
+    /// covers room state change
+    fn room_history_visibility() -> Option<RoomHistoryVisibilityContent>;
+
+    /// covers room state change
+    fn room_join_rules() -> Option<RoomJoinRulesContent>;
+
+    /// covers room state change
+    fn room_name() -> Option<RoomNameContent>;
+
+    /// covers room state change
+    fn room_pinned_events() -> Option<RoomPinnedEventsContent>;
+
+    /// covers room state change
+    fn room_power_levels() -> Option<RoomPowerLevelsContent>;
+
+    /// covers room state change
+    fn room_server_acl() -> Option<RoomServerAclContent>;
+
+    /// covers room state change
+    fn room_third_party_invite() -> Option<RoomThirdPartyInviteContent>;
+
+    /// covers room state change
+    fn room_tombstone() -> Option<RoomTombstoneContent>;
+
+    /// covers room state change
+    fn room_topic() -> Option<RoomTopicContent>;
+
+    /// covers room state change
+    fn space_child() -> Option<SpaceChildContent>;
+
+    /// covers room state change
+    fn space_parent() -> Option<SpaceParentContent>;
```

Comment:
can't mark all here but from a API usage perspective, I'd rather have the functions of 1037-1095 on a separate `RoomStateEvent`-wrapper object which could also hand out for `Activity` then.